__tests__/app.test.js

```diff
@@ -95,4 +95,46 @@ describe('/api/articles', () => {
             expect(articles).toBeSortedBy('created_at', {descending: true})
         })
     })
-})
\ No newline at end of file
+})
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('200: adds a comment with corresponding article_id', () => {
+        const testComment = {
+            username: 'butter_bridge',
+            body: 'Test comment',
+        };
+        return request(app)
+            .post('/api/articles/1/comments')
+            .send(testComment)
+            .expect(201)
+            .then((response) => {
+                expect(response.body.comment.author).toBe(testComment.username);
+                expect(response.body.comment.body).toBe(testComment.body);
+            });
```

Comment:
I'd also add an expect here to check that the correct article id has been returned on the object.